tap_freshdesk/client.py

```diff
@@ -148,7 +148,12 @@ def check_access_token(self):
         """
         Check if the access token is valid.
         """
-        self.request(self.base_url+"/api/v2/roles", {"per_page": 1, "page": 1})
+        try:
+            self.request(self.base_url+"/api/v2/tickets/1/time_entries", {"per_page": 1, "page": 1})
+        except FreshdeskAccessDeniedError:
+            LOGGER.warning("The `Surveys` and the `Timesheets` features are not supported in your current plan. "\
+                            "So, Data collection cannot be initiated for satisfaction_ratings and time_entries streams. "\
+                            "Please upgrade your account to `Pro` plan to use it.")
```

Comment:
it is not the only reason so we should write this in might context like this might be the reason so check your account.

Reply:
Updated `warning` message.